manageiq-operator/Dockerfile

```
@@ -1,5 +1,16 @@
 # Build the manager binary
-FROM docker.io/library/golang:1.23 as builder
+FROM registry.access.redhat.com/ubi9/ubi-minimal as builder
+
+COPY go.mod /go.mod
+
+RUN microdnf -y install gzip jq tar && \
+    arch=$(uname -m) && \
+    if [ ${arch} == "x86_64" ]; then arch="amd64"; fi && \
```

Not saying to do this, but we can trim up this line into a single line with

```suggestion
    arch=$(uname -m | sed 's/x86_64/amd64/') && \
```

Thoughts?

---

Less is less ✂️ ✂️ ✂️